api/src/main/java/io/minio/ObjectConditionalReadArgs.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * MinIO Java SDK for Amazon S3 Compatible Cloud Storage,
+ * (C) 2019 MinIO, Inc.
```

Comment:
```suggestion
 * (C) 2020 MinIO, Inc.
```

Reply:
This file is there since 2019, now added 2020

Reply:
done